modules/module2/module2-08-true_or_false_facets.qmd

```diff
@@ -115,13 +115,16 @@ import altair as alt
 import pandas as pd
 from utils import assert_chart_equal 
 
-def strip_bin(spec):
+def remove_keys_inplace(spec, keys):
     if isinstance(spec, dict):
-        return {k: strip_bin(v) for k, v in spec.items() if k != "bin"}
+        for key in list(spec.keys()):
+            if key in keys:
+                del spec[key]
+            else:
+                remove_keys_inplace(spec[key], keys)
     elif isinstance(spec, list):
-        return [strip_bin(item) for item in spec]
-    else:
-        return spec
+        for item in spec:
+            remove_keys_inplace(item, keys)
```

Comment:
If you are using this in multiple places, could we move it to `utils.py` and just import it here?